sublime_linter.py

```diff
@@ -152,6 +153,19 @@ def on_activated_async(self, view):
         if not util.is_lintable(view):
             return
 
+        # check if the view has been renamed
+        old_filename = buffer_filenames.get(view.buffer_id())
+        if has_filename_changed(view):
+            # update the error store and the views
+            new_filename = util.get_filename(view)
+            if old_filename in persist.file_errors:
+                persist.file_errors[new_filename] = persist.file_errors.pop(old_filename)
+
+            events.broadcast('renamed_file', {
+                'new_filename': new_filename,
+                'old_filename': old_filename
+            })
```

Comment:
Since we update our store, we should ideally just use `update_buffer_errors` but 'updated_error_positions' violates this already. So this is only a reminder that we might need to change something here in the future.